<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddCustomer</class>
 <widget class="QDialog" name="DialogAddCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ajouter un client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/FactDev</normaloff>:/icons/FactDev</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Tél. fixe</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckName" name="leFirstNameReferent">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Roger</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Sauvegarder</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/ok_valid.png</normaloff>:/icons/img/ok_valid.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/cancel.png</normaloff>:/icons/img/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Tél. mobile</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Fax</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckPhone" name="leMobilePhone">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>06 01 02 03 04</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckEmail" name="leEmail">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>roger.balaka@ag42plus.com</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckPhone" name="lePhone">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>05 01 02 03 04</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckName" name="leLastNameReferent">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>BALAKA</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Adresse</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="4">
       <widget class="Gui::Widgets::CheckFields::CheckUntilField" name="leAddress">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>51 Avenue du Général de Gaulle</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ville</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="Gui::Widgets::CheckFields::CheckCity" name="leCity">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Toulouse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Code Postal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Gui::Widgets::CheckFields::CheckPostalCode" name="lePostalCode">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>31400</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Pays</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>N°, Rue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="4">
       <widget class="Gui::Widgets::CheckFields::CheckCountry" name="leCountry">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>France</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Complément</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QLineEdit" name="leComplement">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Bat A - Appt 42</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Société</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckPhone" name="leFax">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>05 01 02 03 09</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckValidField" name="leCompany">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Nom de la société du client</string>
     </property>
     <property name="placeholderText">
      <string>AG42PLUS</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="Gui::Widgets::BrowseImageWidget" name="wgtLogo" native="true">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Site Web</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Gui::Widgets::CheckFields::CheckWebsite" name="leWebsite">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>http://ag42plus.com</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckUntilField</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkuntilfield.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckEmail</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkemail.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckPostalCode</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkpostalcode.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckCity</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkcity.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckPhone</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkphone.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::BrowseImageWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/browseimagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckWebsite</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkwebsite.h</header>
   <container>1</container>
   <slots>
    <signal>textChanged()</signal>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckName</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkname.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckValidField</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkvalidfield.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::CheckFields::CheckCountry</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkcountry.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leLastNameReferent</tabstop>
  <tabstop>leFirstNameReferent</tabstop>
  <tabstop>leCompany</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>lePostalCode</tabstop>
  <tabstop>leCity</tabstop>
  <tabstop>leCountry</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>lePhone</tabstop>
  <tabstop>leMobilePhone</tabstop>
  <tabstop>leFax</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>released()</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>released()</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastNameReferent</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leLastNameReferent</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstNameReferent</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leFirstNameReferent</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCompany</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leCompany</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leAddress</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePostalCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lePostalCode</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCity</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leCity</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEmail</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leEmail</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leMobilePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leMobilePhone</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>414</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFax</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leFax</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCountry</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leCountry</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastNameReferent</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>453</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstNameReferent</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCompany</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePostalCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCity</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCountry</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEmail</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leMobilePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFax</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lePhone</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leWebsite</sender>
   <signal>textChanged(QString)</signal>
   <receiver>DialogAddCustomer</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leWebsite</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leWebsite</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkFields()</slot>
 </slots>
</ui>
